python/ray/util/spark/cluster_init.py

```diff
@@ -1213,19 +1241,20 @@ def setup_ray_cluster(
     cluster.
 
     Args:
-        num_worker_nodes: This argument represents how many ray worker nodes to start
-            for the ray cluster.
-            If autoscale=True, then the ray cluster starts with zero worker node,
-            and it can scale up to at most `num_worker_nodes` worker nodes.
-            In non-autoscaling mode, you can
-            specify the `num_worker_nodes` as `ray.util.spark.MAX_NUM_WORKER_NODES`
+        max_worker_nodes: This argument represents maximum ray worker nodes to start
+            for the ray cluster. you can
+            specify the `max_worker_nodes` as `ray.util.spark.MAX_NUM_WORKER_NODES`
             represents a ray cluster
             configuration that will use all available resources configured for the
             spark application.
             To create a spark application that is intended to exclusively run a
             shared ray cluster in non-scaling, it is recommended to set this argument
             to `ray.util.spark.MAX_NUM_WORKER_NODES`.
-
+        min_worker_nodes: Minimal number of worker nodes in autoscaling mode.
+            if "max_worker_nodes" value is equal to "min_worker_nodes" argument,
+            or "min_worker_nodes" argument is not set, then autoscaling is disabled
```

Comment:
What should be the default value of min_worker_nodes, 0 or max_worker_nodes?

Reply:
default value is equal to `max_worker_nodes` (i.e. default non-autoscaling)